app/routes/forecast_risk_blueprint.py

```diff
@@ -4,4 +4,4 @@
 
 bp = Blueprint("bp_forecast_risk", __name__, "/forecast-risk")
```

Comment:
Needs to add an `url_prefix="/forecast-risk"` in here, i've probably forgot when creating this route.

Reply:
Done.